proxy/core/src/main/scala/io/cloudstate/proxy/ProtobufGeneratedMessageRegisterFeature.scala

```diff
@@ -0,0 +1,32 @@
+package io.cloudstate.proxy
+
+import com.oracle.svm.core.annotate.AutomaticFeature
+import org.graalvm.nativeimage.hosted.Feature.QueryReachabilityAccess
+import org.graalvm.nativeimage.hosted.{Feature, RuntimeReflection}
+
+import scala.collection.JavaConverters._
+
+@AutomaticFeature
+final class ProtobufGeneratedMessageRegisterFeature extends Feature {
+
+  override def duringAnalysis(access: Feature.DuringAnalysisAccess) =
+    List(
+      "akka.protobuf.GeneratedMessage",
+      "akka.protobuf.GeneratedMessage$Builder",
+      "akka.protobuf.ProtocolMessageEnum",
+      "com.google.protobuf.GeneratedMessageV3",
+      "com.google.protobuf.GeneratedMessageV3$Builder",
+      "com.google.protobuf.ProtocolMessageEnum",
+      "scalapb.GeneratedMessage"
+    ).foreach { className =>
+      reachableSubtypes(access, access.findClassByName(className)).foreach { cls =>
```

Comment:
You get `null`, which is then handled in `reachableSubtypes` (which was already handling because in the call graph of `access.reachableSubtypes` it can throw NPE).

Reply:
I'll make it more direct.